app/controllers/concerns/hyrax/controller.rb

```diff
@@ -26,11 +28,6 @@ def default_url_options
     super.merge(locale: I18n.locale)
   end
 
-  # Override Blacklight to use the Hyrax::SearchState
```

Comment:
This method already exists in Blacklight, so we're not removing a public method, just this implementation of it.